doc/audit/builtin_maps.rst

```diff
@@ -128,7 +128,7 @@ Versions of the code allowed to join the current network.
 
 **Value** Represented as JSON.
 
-.. doxygenenum:: ccf::CodeStatus
+.. doxygenstruct:: ccf::CodeInfo
```

Comment:
I think it's the first time we do this (change the value of a type, as opposed to add a new field). Stating that the value here is always a CodeInfo represented as JSON is a stretch, the old ones are are a CodeStatus.

I think we can do one of two things:

1. Leave the CodeStatus type in, and document one before the version, and the other after (literally)
2. Remove the old type, but document it inline here, with the version at which it was removed.

Someone writing an script that audits the ledger is (possibly) going to want to know about this.

Reply:
Added a note below stating old version